src/common/RelatedFunds/RelatedFunds.js

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
+
+import ConnectionListing from '../../components/ConnectionListing';
+
+const RelatedFunds = ({ parentMutator, query, history, fundsToDisplay, currency }) => {
+  const openFund = useCallback(
+    (e, fund) => {
+      const path = `/finance/fund/view/${fund.id}`;
+
+      history.push(path);
+    },
+    [history],
+  );
+
+  useEffect(() => {
+    parentMutator.groupFundFiscalYears.reset();
+    parentMutator.groupFundFiscalYears.GET({
+      params: {
+        limit: LIMIT_MAX,
+        query,
+      },
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [query]);
+
```

Comment:
Could you add a condition for showing loader during data is loading?

Reply:
Added